pebblo/reports/templates/weasyprintTemplate.html

```diff
@@ -82,6 +82,7 @@
             Top Documents With Most Findings
           </div>
           <div class="mt-3 inter">
+            {% if data.topFindings|length > 0 %}
```

Comment:
Lets use same condition that we used for local UI for this empty state.

Reply:
Updated condition